cps/main.c

```diff
@@ -782,8 +788,19 @@ cps_stage1(void *arg)
 		goto error;
 	}
 
+	cps_conf->front_kni_index = if_nametoindex(name);
```

Comment:
When we call `rte_eth_dev_stop()` and `rte_eth_dev_start()` in `kni_change_if()`, is it possible for the index to change? If so, `kni_change_if()` needs to update `cps_conf->front_kni_index`.

This comment also applies to `cps_conf->back_kni_index` below.

Reply:
I don't think it's possible for the index to change just from starting/stopping the device, since you're supposed to be able to listen on an interface's index to get changes like this from the kernel. I can't find anything in the documentation about when interface indices can be changed, though.